assets/src/edit-story/components/canvas/displayElement.js

```diff
@@ -51,13 +53,30 @@ const BackgroundOverlay = styled.div`
   top: 0;
   left: 0;
 `;
+
 const ReplacementContainer = styled.div`
   transition: opacity 0.25s cubic-bezier(0, 0, 0.54, 1);
   pointer-events: none;
   opacity: ${({ hasReplacement }) => (hasReplacement ? 1 : 0)};
 `;
 
-function DisplayElement({ element, previewMode, page }) {
+function AnimationWrapper({ children, id, animationMode }) {
+  const enableAnimation = useFeature('enableAnimation');
```

Comment:
What if we enable/disable animations at the `AnimationPart` level?  This way whether it's an AMP story or a dashboard story or anything with one of our animations in it, it'll be nipped in the bud with just one flag.  I think we can do this if we updated our `AnimationPart` lookup code to something like this:
```
export function AnimationPart(type, args) {
  if (!useFeature('enableAnimation')) {
    return throughput();
  }
  
  const generator =
    {
      [ANIMATION_TYPES.BOUNCE]: AnimationBounce,
    }[type] || throughput;

  return generator(args);
}
```

This way, we'll just return a bunch of pass through wrap arounds and nothing will be animated nor extra containers generated.  Would this work?

Reply:
Also, if this makes sense to you can you rebase in the latest master?  The `AnimationPart` in this repo is out of date.

Reply:
yea good call. will do